From 6db78200993986d8b3c6a2cd1d6ba9d7660f185f Mon Sep 17 00:00:00 2001
From: Julien Olivain <juju@cotds.org>
Date: Sun, 17 Jan 2016 21:08:53 +0100
Subject: [PATCH 7/8] gl_painter.hpp: handle OpenGL ES case

Signed-off-by: Julien Olivain <juju@cotds.org>
---
 src/video/gl/gl_painter.hpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/video/gl/gl_painter.hpp b/src/video/gl/gl_painter.hpp
index bb3dd75..423e7ae 100644
--- a/src/video/gl/gl_painter.hpp
+++ b/src/video/gl/gl_painter.hpp
@@ -21,10 +21,15 @@
 #include <glbinding/gl/gl.h>
 using namespace gl;
 #else
+
+#ifdef HAVE_OPENGLES
+#include "SDL_opengles.h"
+#else
 #include <GL/glew.h>
 
 #include "SDL_opengl.h"
 #endif
+#endif
 
 struct DrawingRequest;
 
-- 
2.5.0

